modules/credit-card-types.ts

```diff
@@ -0,0 +1,172 @@
+// Adapted from https://www.npmjs.com/package/credit-card-type
+
+export declare type CreditCardBrandId =
+  | 'american-express'
+  | 'diners-club'
+  | 'discover'
+  | 'jcb'
+  | 'mastercard'
+  | 'unionpay'
+  | 'visa';
+declare type CreditCardBrandName =
+  | 'American Express'
+  | 'Diners Club'
+  | 'Discover'
+  | 'JCB'
+  | 'Mastercard'
+  | 'UnionPay'
+  | 'Visa';
+declare type CreditCardSecurityCodeLabel = 'CVV' | 'CVC' | 'CID' | 'CVN';
+declare type Range = [number, number];
+declare type CreditCardPattern = number | Range;
+export declare type CreditCardInformation = {
+  niceType: CreditCardBrandName;
```

Comment:
Pas fan de ce nom de clé, alors qu'il s'agit du `name`:
```suggestion
  name: CreditCardBrandName;
```

Reply:
Effectivement c'est un oubli quand j'ai repris certains types de la librairie `credit-card-type`